src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -37,31 +37,31 @@ const ReportsPageRanking = ({ domainName, dateFrom, dependencies: { datahubClien
       {state.loading ? (
         <Loading />
       ) : (
-        <div className="reports-box">
+        <S.ReportBox>
           <small className="title-reports-box">
             <FormattedMessage id="reports_pageranking.top_pages" />
           </small>
 
           {state.pages ? (
             state.pages.map((item, index) => (
-              <StyledPageRankingItem key={index}>
-                <div>
-                  <StyledParagraph>
+              <S.PageListItem key={index}>
```

Comment:
Why `PageListItem`? I thought we agreed styled components will have semantic visual meaning :/ Perhaps just `ListItem`?

Reply:
Yes, i can change